targets/unity-v2/source/Shared/Public/PlayFabAuthenticationContext.cs

```diff
@@ -2,33 +2,59 @@ namespace PlayFab
 {
     public sealed class PlayFabAuthenticationContext
     {
-        public string PlayFabId;
-
         public PlayFabAuthenticationContext()
         {
         }
 
-        public PlayFabAuthenticationContext(string clientSessionTicket, string entityToken, string playFabId) : this()
+        public PlayFabAuthenticationContext(string clientSessionTicket, string entityToken, string playFabId, string entityId, string entityType) : this()
```

Comment:
>PlayFabAuthenticationContext [](start = 15, length = 28)

You are breaking people here. Make two last parameters null by default

Reply:
It's my opinion that this specific constructor is meant for internal use.
Customers won't ever provide those parameters, because they can't know them in advance.

Reply:
(Maybe that indicates needing to change that constructor to internal...)